Oya-Engineering/sum-two-numbers
# edit wording on readme file

**What does the PR do**
- edit the wording description of the README.md file 

**How can this be tested**
- go to the repository sum-two-numbers and change to the branch chore-add-read-me observe the description.
